Gemfile.lock

```diff
@@ -43,10 +43,14 @@ GEM
     byebug (9.0.6)
     concurrent-ruby (1.0.2)
     erubis (2.7.0)
+    faraday (0.9.2)
+      multipart-post (>= 1.2, < 3)
     ffi (1.9.14)
     globalid (0.3.7)
       activesupport (>= 4.1.0)
+    hashie (3.4.6)
     i18n (0.7.0)
+    jwt (1.5.6)
```

Comment:
that's just the gemfile.lock, the gemfile actually lists dependencies. i was trying out different stuff so the .lock file has some unecessary stuff but it doesn't really matter. it won't be installed when you do a bundle install.